tests/common.go

```diff
@@ -127,6 +127,29 @@ func AddPgExecuteSqlConfig(t *testing.T, config map[string]any) map[string]any {
 	return config
 }
 
+// AddMSSqlExecuteSqlConfig gets the tools config for `postgres-execute-sql`
+func AddMSSqlExecuteSqlConfig(t *testing.T, config map[string]any) map[string]any {
```

Comment:
```suggestion
// AddMssqlExecuteSqlConfig gets the tools config for `mssql-execute-sql`
func AddMssqlExecuteSqlConfig(t *testing.T, config map[string]any) map[string]any {
```

to keep consistency with the other functions.